specs/basic-auth/basic-auth.spec.js

```diff
@@ -1,17 +1,16 @@
-const { DriverProvider } = require('protractor/built/driverProviders')
+
 const BasicAuth = require('../../page-objects/basic-auth')
 
 describe('Given the authentication page', () => {
-    let autheticationPage
-
-    beforeAll(() => {
-        autheticationPage = new BasicAuth()
-    })
+  let autheticationPage
 
-    it('Shows up the authentication', () => {
-        //The Solution to Authentication Pop Up:driver.get(protocol://Usename:Password@URL Address);
-        browser.driver.get('https://admin:admin@the-internet.herokuapp.com/basic_auth') 
-        expect(autheticationPage.successMessage.first().getText()).toBe("Congratulations! You must have the proper credentials.")
-    })
+  beforeAll(() => {
+    autheticationPage = new BasicAuth()
+  })
 
-})
\ No newline at end of file
+  it('Shows up the authentication', () => {
+    // The Solution to Authentication Pop Up:driver.get(protocol://Usename:Password@URL Address);
+    browser.driver.get('https://admin:admin@the-internet.herokuapp.com/basic_auth')
```

Comment:
put this URL in a CONST on top of this class

Reply:
done